testing_and_setup/compass/ocean/drying_slope/forward_template.xml

```diff
@@ -21,8 +21,8 @@
 		<option name="config_time_integrator">'RK4'</option>
 		<option name="config_run_duration">'0000_12:00:01'</option>
 		<option name="config_use_mom_del2">.false.</option>
-		<option name="config_use_const_visc">.true.</option>
-		<option name="config_vert_visc">1e4</option>
+		<option name="config_use_cvmix_background">.true.</option>
+		<option name="config_cvmix_background_viscosity">1e4</option>
```

Comment:
Is this value correct?  It is massive for a vertical viscosity.

Reply:
We can't run single layer so this is a "hack" to get the behavior to match theoretical results in our V&V.